packages/svelte-check/src/index.ts

```diff
@@ -210,10 +242,15 @@ class DiagnosticsWatcher {
         this.scheduleDiagnostics();
     }
 
-    scheduleDiagnostics() {
+    scheduleDiagnostics(updateWatchers = false) {
```

Comment:
I would prefer this to be a separate method like `onProjectReloaded` since updateWatcher is not related to `scheduleDiagnotics`